docs/changelog.md

```diff
@@ -35,6 +35,9 @@ permalink: /changelog/
   * Added `CustomTabState.trustedOrigins` extension method to turn the verification state of a custom tab into a list of origins.
   * Added `WebAppHideToolbarFeature.onTrustedScopesChange` to change the trusted scopes after the feature is created.
 
+* **service-telemetry**
+  * This component is now deprecated. Please use the [Glean SDK](https://mozilla.github.io/glean/book/index.html) instead.
+
```

Comment:
Let's also mention what you wrote in the README here: That we are not removing this until apps are migrated.